src/main/java/com/parasoft/demoapp/graphql/ItemGraphQLDataFetcher.java

```diff
@@ -82,4 +80,24 @@ public DataFetcher<PageInfo<ItemEntity>> getItems() {
             }
         };
     }
+
+    public DataFetcher<ItemEntity> getItemByName() {
+        return dataFetchingEnvironment -> {
+            try {
+                Map<String, String> uriVariables = new HashMap<>();
+                if (dataFetchingEnvironment.containsArgument("itemName")) {
+                    uriVariables.put("itemName", dataFetchingEnvironment.getArgument("itemName"));
+                }
```

Comment:
Please change this block to be:
        String itemName = dataFetchingEnvironment.getArgument("itemName");
        if (itemName  != null && !itemName.isEmpty()) {
            uriVariables.put("itemName", itemName);
        }